fibbingnode/southbound/lsdb.py

```diff
@@ -466,7 +505,30 @@ def process_lsa(self):
                                  for part in lsa_info.split(SEP_GROUP) if part]
                     hdr = LSAHeader.parse(lsa_parts.pop(0))
                     lsa = LSA.parse(hdr, lsa_parts)
+                    new_seqnum = hdr.lsa_seqnum
+
                     log.debug('Parsed %s: %s', action, lsa)
+
+                    # For now, we just log lsa seq_num and ignore an LSA when
+                    # it has an older seq_num than the one in the database
+                    if hdr.lsa_type == '5':
```

Comment:
When I log seq_num for all LSA types, and ignore those LSA received with an older seq_num than the one currently in the database... it seems that some of the R and N lsas arrive with an old seq_num, which 

`[  DEBUG] endpoints: Cannot resolve network lsa for 192.168.190.2 yet
[  DEBUG] process_lsa: Parsed REM: [N]<172.16.0.1: 172.16.0.1, 172.16.0.2>
[  DEBUG] process_lsa: Current seqnum for LSA in lsdb: -2147483641)
[  DEBUG] process_lsa: OLD seqnum for LSA: [N]<172.16.0.1: 172.16.0.1, 172.16.0.2>, ignoring it...
[  DEBUG] process_lsa: Parsed ADD: [N]<172.16.0.1: 172.16.0.1, 172.16.0.2, 172.16.0.3>
[  DEBUG] process_lsa: Current seqnum for LSA in lsdb: -2147483641)
[  DEBUG] process_lsa: OLD seqnum for LSA: [N]<172.16.0.1: 172.16.0.1, 172.16.0.2, 172.16.0.3>, ignoring it...`

Which makes these LSAs unresolvable:

`[  DEBUG] endpoints: Cannot resolve network lsa for 172.16.0.1 yet
[  DEBUG] endpoints: Cannot resolve network lsa for 172.16.0.1 yet`